schedulingApp/templates/sections.html

```diff
@@ -0,0 +1,38 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sections</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet"
+          integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous">
+    {% load static %}
+    <link rel="stylesheet" type="text/css" href="{% static 'global.css' %}"/>
+</head>
+<body>
+<div class="container-fluid">
+    <div class="row">
+        {% include 'navbar.html' %}
+        <div class="col-sm-10">
+            <h1>Active Sections</h1>
+            <table>
```

Comment:
This looks good; will be updating this later today with improved Bootstrap table code.